src/main/java/pro/sky/java/school/Controller/ExamController.java

```diff
@@ -0,0 +1,22 @@
+package pro.sky.java.school.Controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.java.school.Service.ExaminerService;
+
+@RestController
+@RequestMapping("/exam/java/exam")
+public class ExamController {
+
+    ExaminerService examinerService;
```

Comment:
```suggestion
    private ExaminerService examinerService;
```
не забываем про модификаторы доступа